app/services/grant_submission_submission_services/new.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module GrantSubmissionSubmissionServices
+  module New
+    def self.call(submission:, applicant:)
```

Comment:
Is the `applicant` argument necessary here? A new submission is being submitted and the newly created applicant can be retrieved from `submission.submitter`

Reply:
I took this out. I think it is not necessary now, but in the future, we will probably want to add something back in for setting a non-submitter as applicant on submission create.